HISTORY.rst

```diff
@@ -2,6 +2,11 @@
 History
 =======
 
+Current
```

Comment:
I thinks it's more clean and straight forward if you change the version just before the release because as more PRs will come, each release will contain more new functionality and is easier to track the differences between the master branch (the new release) and the latest release. 

At least that's what I saw all the big projects do 😄 (django, tornado, etc): the core contributors are responsible for the release cycle 😛